### PR TITLE
[Frontend] some minor fixes to make it run

### PR DESCRIPTION
**Pull request by mustansirMuz**
Asana link: https://app.asana.com/0/1204411152293714/1204600474221107/f

- Made the frontend runnable by making a minor fix, specifically fixing an import in the BookDelete.tsx file

### CODE REVIEW

File: frontend/src/components/pages/BookDelete.tsx [hunk 1]
```diff
@@ -9,35 +9,33 @@ import { BookDetailsContainer } from "./BookDetailsContainer";
...
+      {/* Displaying the book details component to show which book is being deleted */}
+      <BookDetailsContainer showLinks={false} />
+      <button onClick={handleClick}>Yes, Delete</button>
+    </div>
+  );
 }
 
-export default BookDelete;
\ No newline at end of file
+export default BookDelete;
```
Comment (mtalhatahir2001) [inline frontend/src/components/pages/BookDelete.tsx:41]: Empty line is missing at the end of file.

**Review by mtalhatahir2001 — DISMISSED** [review]

File: frontend/src/components/pages/BookDelete.tsx (hunk 1 above)
Comment (Sana-Ullah786) [reply frontend/src/components/pages/BookDelete.tsx:41]: Yes Same Request.

**Review by Sana-Ullah786 — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/BookDelete.tsx (hunk 1 above)
Comment (Sana-Ullah786) [reply frontend/src/components/pages/BookDelete.tsx:41]: Okay

**Review by Sana-Ullah786 — APPROVED** [review]
Approved. ✨

**Review by SohaibkhanFolio3 — APPROVED** [review]

**Pull request merged**